app/src/main/res/menu/feedlist.xml

```diff
@@ -16,20 +16,19 @@
         android:title="@string/filter"
         custom:showAsAction="always">
     </item>
-    <item
-        android:id="@+id/refresh_item"
-        android:icon="@drawable/ic_refresh"
-        android:menuCategory="container"
-        android:title="@string/refresh_label"
-        custom:showAsAction="always">
-    </item>
     <item
         android:id="@+id/refresh_complete_item"
         android:menuCategory="container"
         android:title="@string/load_complete_feed"
         custom:showAsAction="collapseActionView">
     </item>
 
+    <item
+        android:id="@+id/refresh_item"
+        android:icon="@drawable/ic_refresh"
+        android:menuCategory="container"
+        android:title="@string/refresh_label"
+        custom:showAsAction="always"></item>
```

Comment:
Why did you re-order the items? Now force-full-refresh is displayed before the normal refresh button.

Reply:
dont know, probably by accident